nfsstat/assets/configuration/spec.yaml

```
@@ -0,0 +1,19 @@
+name: Nfsstat
+files:
+- name: nfsstat.yaml
+  options:
+  - template: init_config
+    options:
+    - name: nfsiostat_path
+      description: |
+        A version of nfsiostat is shipped with the Agent binary,
+        but if you are using a source install, specify the location of nfsiostat
```

```suggestion
        A version of nfsiostat is shipped with the Agent binary.
        If you are using a source install, specify the location of nfsiostat
```
nit, easier to read?